Dockerfile

```diff
@@ -2,20 +2,18 @@
 # supports Debian 11
 FROM python:3.8-slim-buster AS build-image
 
-RUN apt update
-RUN apt full-upgrade -y
-RUN apt install -y --no-install-recommends \
+RUN apt-get update
+RUN apt-get install -y --no-install-recommends \
```

Comment:
`apt` is the newer command, it's not just meant for users, it's perfectly fine to use in scripts too. I don't see a reason not to use it.

Reply:
https://manpages.debian.org/bullseye/apt/apt.8.en.html :

> apt provides a high-level commandline interface for the package management system. It is intended as an end user interface and enables some options better suited for interactive usage by default compared to more specialized APT tools like apt-get(8) and apt-cache(8).

Reply:
Fair enough, they give the same result in practice, but I guess using apt-get avoids the warning in scripts.

I still want to keep the `full-upgrade`, since upgrading can bring in security fixes.

Reply:
Official `python` docker image most probably has already latest security fixes ASAP.
Also building different docker images using the same Dockerfile should be prevented whenever possible (consider a situation when you have build a docker image and someone else builds this a week later) , and if you build an image right now and do not remove your docker images, and try building the same docker image, docker will skip this step anyway.

Reply:
However if you want to I can whip up another PR which builds docker and runs trivy vulnerability scanner to make sure there is not any HIGH severity security bug which has been fixed in a newer version present in docker image.

Reply:
That's kinda the point. If you pull the python image now and in a week, you might get a different base image as it might have been updated in the meantime. If you want exact images, you'd have to use the hash.
Given that you might get a new image anyway, might as well ensure you always get the latest version for all relevant packages.

Reply:
Fair. I will revert the commit.

Reply:
Dropped the commit and force-pushed.
https://github.com/albertodonato/query-exporter/compare/81ff8a84dd8bea5409a3914ab4dd4c9388db0598..6b17cd06b2f7ea6bd346a047a17db63a1aad9122